README.md

```diff
@@ -124,6 +124,9 @@ The keys are:
   "localhost"; so pointing to your local caching resolver using `127.0.0.1`
   will not end in happiness and puppies.
 
+* **`host_hostname`**: the hostname to use when generating the container
+  hostname.
```

Comment:
It would be helpful to explain the circumstances under which this might be useful.